documentation_builder/simulating.ipynb

```diff
@@ -758,13 +759,14 @@
       "PRODUCING REACTIONS -- Pyruvate\n",
       "-------------------------------\n",
       "  %             FLUX   RXN ID                        REACTION                       \n",
-      " 50.0%   6.13 ± 6.13      PYK                  adp_c + h_c + pep_c --> atp_c + pyr_c\n",
-      " 50.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
+      " 85.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
```

Comment:
You could also do center of min, max
On Mar 16, 2016 12:24, "Peter St. John" <notifications@github.com> wrote:

> In documentation_builder/simulating.ipynb
> <https://github.com/opencobra/cobrapy/pull/239#discussion_r56398731>:
>
> > @@ -758,13 +759,14 @@
> >        "PRODUCING REACTIONS -- Pyruvate\n",
> >        "-------------------------------\n",
> >        "  %             FLUX   RXN ID                        REACTION                       \n",
> > -      " 50.0%   6.13 ± 6.13      PYK                  adp_c + h_c + pep_c --> atp_c + pyr_c\n",
> > -      " 50.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
> > +      " 85.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
>
> True, I used half-spans +/- range/2 because its a familiar format.
> Probably better to make summary just have "MAX" and "MIN" columns when
> called with FVA.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencobra/cobrapy/pull/239/files/8a60f16e28ddb40995a91b00b3b74e188f8eb93b#r56398731>
>